docs/catalyst_101.md

```diff
@@ -319,49 +318,50 @@ All meetings have pre-designated facilitators and note-takers that are specified
-
-**Attendees:** Appointed person(s) or volunteer(s)
-**Notes:** n/a
-
-- Catalyst participates in weekly Fossil Fuel Refinance Project calls. These calls are a consortium of energy policy folks working on refinancing strategies to close coal plants and other fossil fuel-driven generation assets. Usually one-two Catalyst members join these calls.
-- For the call, you’ll need to solicit updates from the Catalyst team in order to communicate our weekly progress to the refinance team; coordinating collaboration with the refinance team; and taking notes on relevant updates from the call to convey to the Catalyst team.
+- Board meetings may be cancled if there is nothing on the agenda.
 
 (contractor-feedback-meeting)=
 
 #### 3 and 6-month Contractor Feedback (two 1 hour blocks)
 
-**Attendees:** Members and contractor being reviewed
+**Attendees:** Members and contractor being reviewed.
 **Notes:** [Feedback for Contractor](https://drive.google.com/open?id=1IVZ6sgM7NkMG09o4pVwcUf5bVz5PUVrD)
 
 - Catalyst contractors will participate in a 3-month check-in and a 6-month check-in after which the members will vote on whether to invite them to become a member of the co-op.
-
 - These check-ins are preceded by written feedback from the contractor about Catalyst and the members about the contractor pursuant to a form distributed a week before the feedback session. The first hour block is dedicated to co-op feedback for the contractor while the second hour block is dedicated to the contractor’s feedback for the co-op. These meetings may be back to back or scheduled separately a few days or hours apart.
 
 (annual-member-feedback)=
 
 #### Annual Member Feedback (two 1 hour blocks)
 
-**Attendees:** Appointed members and member being reviewed (reviewee)
+**Attendees:** Appointed members and member being reviewed (reviewee).
 **Notes:** [Member Feedback](https://drive.google.com/open?id=1zsKiLe1hgV3vuPOxuQZ7JljkRpj-PSOq)
 
 - Catalyst members participate in annual check-ins where they have the opportunity to review the co-op as well as be reviewed by other members.
-
 - These check-ins are preceded by written feedback from the reviewee about Catalyst and the other members about the reviewee pursuant to a form distributed a week before the feedback session. The first hour block is dedicated to co-op feedback for the reviewee while the second hour block is dedicated to the reviewee’s feedback for the co-op. These meetings may be back to back or scheduled separately a few days or hours apart.
 
+(monthly-member-check-ins)=
+
+#### Monthly Member Check-ins (30 min)
+
+**Attendees:** Member and check-in facilitator.
+**Notes:** Notes are private and only shared with the member and meeting facilitator.
+
+- Catalyst members participate in monthly check-in calls where they
+  touch base on their annual goals and general feelings. These calls are intended to provide members with an outlet for expressing excitement or concern about anything going on.
```

Comment:
```suggestion
  touch base on their annual goals and general feelings. These calls are intended to help members keep track of their goals and make sure they are supported in achieving them.
```

Though you are right in that a lot of the support turns out to be providing an outlet to talk about work feels.